plugins/rtp/src/plugin.vala

```diff
@@ -276,7 +276,6 @@ public class Dino.Plugins.Rtp.Plugin : RootInterface, VideoCallPlugin, Object {
             device_monitor.stop();
         }
         destroy_call_pipe();
-        Gst.deinit();
```

Comment:
I should have added more info about this in the commit message. The reason I removed it was because it caused the Dino proces to linger in the background when (indeed like you say) I was using Gst in the ringer plugin. BUT my reasoning for leaving it out is that this is a nasty bug that might crop up again in the future when someone tries to do anything else at all with Gst. 

And so I did the following:
+ checked this https://gstreamer.freedesktop.org/documentation/gstreamer/gst.html#gst_deinit
+ tested that Dino works without the Gst.deinit
+ tried looking for a specific reason for the deinit with ```git log -L 276,+10 -- plugins/rtp/src/plugin.vala``` and didn't find anything

All this made me conclude that it is better to leave it out but if I might be missing something or if you realy want to keep it "just in case", ofcourse let me know and I will add it back. (and maybe add a comment+commit msg warning with the stuff I am talking about here)

Reply:
All right, just please put this in a separate commit and include your reasoning in the commit message.